templates/index.html

```diff
@@ -1,56 +1,283 @@
-    <meta charset="UTF-8" />
-
-    <!-- Add meta name below -->
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    
-    <!-- Add page title below -->
-    <title>Charmed OSM</title>
-    
-    <!-- Add preconnect/dns-prefetch below -->
-    <link rel="preconnect" href="https://assets.ubuntu.com">
-    <!-- <link rel="dns-prefetch" href="http://www.spreadfirefox.com/"> -->
-    
-    <!-- Add inline JS below -->
-    <script>// Inline JS</script>
-    
-    <!-- Add stylesheet below -->
-    <link rel="stylesheet" type="text/css" media="screen" href="/static/css/styles.css" />
-    
-    <!-- Add inline CSS below -->
-    <style>/* Inline CSS */</style>
-    
-    <!-- Add preload below -->
-    <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/e8c07df6-Ubuntu-L_W.woff2" crossorigin>
-    <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/7f100985-Ubuntu-Th_W.woff2" crossorigin>
-    <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/f8097dea-Ubuntu-LI_W.woff2" crossorigin>
-    <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/fff37993-Ubuntu-R_W.woff2" crossorigin>
-
-    <!-- Add script below -->
-    <!-- <script src=""></script> -->
-
-    <!-- Add prefetch/prerender below -->
-    <!-- <link rel="prefetch" href="/style.css" as="style" />
-    <link rel="prerender" href="https://example.com/about.html" /> -->
-  </head>
-
-  <body>
-    <!-- Add navigation below -->
-
-    <main>
-      <!-- Add main content below -->
-      <div class="p-strip--dark">
-        <div class="u-fixed-width">
-          <h1>Charmed OSM</h1>
-        </div>
+{% block content %}
+<section class="p-strip--header is-dark">
+  <div class="row u-vertically-center u-equal-height">
+    <div class="col-8">
+      <h1>
+        NFV Management and Orchestration (MANO)
+      </h1>
+      <p class="p-heading--four u-sv3">
+        Accelerate your migration to NFV with Open Source MANO (OSM)
+      </p>
+      <p>
+        Automate deployment and operations of network functions, and benefit from reduced operational costs, by using an open source implementation of ETSI NFV MANO. To improve the stability of deployments and provide telcos with a production-grade platform, Canonical delivers a pure upstream OSM distribution &mdash; Charmed OSM. As a key contributor to the OSM project, we are excited to help you on your NFV journey.
+      </p>
+      <p><a href="/contact-us" class="p-button--positive">Get in touch</a></p>
+      <p><a href="https://tutorials.ubuntu.com/tutorial/charmed-osm-get-started#0" class="p-link--inverted">Get Started with Charmed OSM&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-4 u-align--center u-hide--small">
+      <img src="https://assets.ubuntu.com/v1/e95779ea-opensource-MANO-white.svg" width="300" height="109"
+        alt="Open Source Mano logo">
```

Comment:
Open Source Mano logo … what? This is a decoration, and doesn’t link to anything, so:
```suggestion
        alt="">
```